emaildrafter/views.py

```diff
@@ -0,0 +1,19 @@
+from django.shortcuts import render
+from django.http import HttpResponseRedirect
+from .forms import UserForm
+
+
+def index(request):
+    if request.method == "POST":
+        form = UserForm(request.POST)
```

Comment:
This is good. Is there a reason for using function based view instead of class based?

Reply:
I've started with function based views rather than class based as this emulates the Flask app behaviour so should ease the everyone's transition to the Django way of doing things.

Reply:
Ah okay, makes sense!